### PR TITLE
Bump to latest 19.08 runtime, provide appdata files

### PR DESCRIPTION
**Pull request by adds68**

### CODE REVIEW

File: com.stremio.Stremio.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  <content_rating type="oars-1.1">
```
Comment (bilelmoussaoui) [inline com.stremio.Stremio.appdata.xml:37]: This lacks at least the component tag along with the id, the rest can be added later

File: com.stremio.Stremio.yml [hunk 2]
```diff
@@ -57,7 +61,7 @@ modules:
   - install stremio-wrapper /app/bin/
 
   - mkdir -p /app/share/icons/hicolor/512x512/apps/
-  - cp squashfs-root/stremio.png /app/share/icons/hicolor/512x512/apps/com.stremio.Stremio.png
+  - cp com.stremio.Stremio.svg /app/share/icons/hicolor/512x512/apps/com.stremio.Stremio.svg
```
Comment (bilelmoussaoui) [inline com.stremio.Stremio.yml:64]: Have you tried to build it from source? Last time I tried I needed Qt Web Engine, but it's now available here https://github.com/flathub/io.qt.qtwebengine.BaseApp. 
Of course, we can merge this as it's for now

File: com.stremio.Stremio.appdata.xml (hunk 1 above)
Comment (adds68) [reply com.stremio.Stremio.appdata.xml:37]: I generated this using: https://hughsie.github.io/oars/generate.html

So i assumed that would have everything in there?

File: com.stremio.Stremio.yml (hunk 2 above)
Comment (adds68) [reply com.stremio.Stremio.yml:64]: No i just wanted to get this initial build working and try to publish it, but i don't mind having another go! :)

File: com.stremio.Stremio.appdata.xml (hunk 1 above)
Comment (bilelmoussaoui) [reply com.stremio.Stremio.appdata.xml:37]: Oh, the appdata needs more tags than just the OARS one. The specs are documented here https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps

Comment (adds68) [reply com.stremio.Stremio.appdata.xml:37]: I've updated the appdata fie, i just need your email for the update contact, i can add mine though if you are too busy? :)

File: com.stremio.Stremio.yml (hunk 2 above)
Comment (adds68) [reply com.stremio.Stremio.yml:64]: Once we merge this, i will work on building the source version

File: com.stremio.Stremio.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  <id>com.stremio.Stremio</id>
+  <provides><id>com.stremio.Stremio.desktop</id></provides>
+  <launchable type="desktop-id">com.stremio.Stremio.desktop</launchable>
+  <name>Stremio</name>
+  <developer_name>Stremio</developer_name>
+  <summary>Discover, organize and watch video from all kind of sources on any device that you own.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPLv3</project_license>
+  <url type="homepage">https://www.stremio.com/</url>
+  <url type="help"></url>
```
Comment (bilelmoussaoui) [inline com.stremio.Stremio.appdata.xml]:
```suggestion
  <url type="help"></url>
```
Replace the help url with a bugtracker one, they have one over here https://github.com/Stremio/stremio-shell

File: com.stremio.Stremio.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.stremio.Stremio</id>
+  <provides><id>com.stremio.Stremio.desktop</id></provides>
```
Comment (bilelmoussaoui) [inline com.stremio.Stremio.appdata.xml]:
```suggestion
```
The provides is already added automatically from the launchable tag

File: com.stremio.Stremio.appdata.xml (hunk 1 above)
Comment (bilelmoussaoui) [reply com.stremio.Stremio.appdata.xml:37]: Perfect, sure, you can use mine: bil[dot]elmoussaoui[at]gmail[dot]com. The only missing part is to install the appdata file in `$prefix/$datadir/metainfo`
Btw, can you rename the appdata file to `$app-id.metainfo.xml` ? New apptadata files should use the metainfo extension prefix instead.

**Comment by bilelmoussaoui** [comment]
Thanks @Wetrain :)

**Pull request merged**